server/server.js

```diff
@@ -10,11 +12,15 @@ const app = express();
 //   })
 // );
 
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
 // Simple route
 app.get("/", (req, res) => {
   console.log("Welcome to miah");
   res.json({ message: "Welcome to miah" });
 });
 
+
 // Port 
-app.listen(8000, () => console.log("listening on port 8000"));
\ No newline at end of file
+app.listen(3000, () => console.log("listening on port 3000"));
```

Comment:
Revert this change back to port 8000, it was fine as it was. 
The React app/front-end is on port 3000.